po/chkconfig.pot

```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2024-01-09 12:58+0100\n"
+"POT-Creation-Date: 2024-07-24 11:00+0200\n"
```

Comment:
Spurious change; I think gettext will do this by default unfortunately. Could probably break out "update gettext" into a distinct PR, otherwise this would (in the extremely unlikely event someone did a different PR) create unnecessary conflicts.

Reply:
Will update before (potentially) merging